plugins/chat/assets/javascripts/discourse/components/chat-drawer.js

```diff
@@ -188,11 +188,13 @@ export default Component.extend({
   },
 
   @action
-  openInFullPage() {
+  async openInFullPage() {
     this.chatStateManager.storeAppURL();
     this.chatStateManager.prefersFullPage();
     this.chat.activeChannel = null;
 
+    await new Promise((resolve) => next(resolve));
```

Comment:
the drawer spec was failing and I noticed that when doing minimize/maximize super fast we could end up in a weird state, this should prevent this.